app/models/payment_history/api_importer.rb

```diff
@@ -0,0 +1,154 @@
+require "graphql/client"
+require "graphql/client/http"
+
+class PaymentHistory::ApiImporter
+  include ShopifyPartnerAPI
+
+  THROTTLE_MIN_TIME_PER_CALL = 0.3
+
+  API_REVENUE_TYPES = {
+    "recurring_revenue" => [
+      "AppSubscriptionSale"
+    ],
+    "onetime_revenue" => [
+      "AppOneTimeSale",
+      "ServiceSale",
+      "ThemeSale"
+    ],
+    "affiliate_revenue" => [
+      "ReferralTransaction"
+    ],
+    "refund" => [
+      "AppCredit",
+      "AppSaleAdjustment",
+      "AppSaleCredit",
+      "ReferralAdjustment",
+      "ServiceSaleAdjustment",
+      "ThemeSaleAdjustment"
+    ],
+    "usage_revenue" => [
+      "App sale – usage",
+      "Usage application fee",
+      "AppUsageRecord",
+      "AppUsageSale",
```

Comment:
**[standard]** <Style/TrailingCommaInArrayLiteral> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Avoid comma after the last item of an array.
```suggestion
      "AppUsageSale"
```